src/main/java/ru/skypro/homework/controller/AdsController.java

```diff
@@ -1,23 +1,155 @@
 package ru.skypro.homework.controller;
 
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
 import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
-import ru.skypro.homework.dto.Ad;
-import ru.skypro.homework.dto.Ads;
+import ru.skypro.homework.dto.ads.Ad;
+import ru.skypro.homework.dto.ads.Ads;
+import ru.skypro.homework.dto.ads.CreateOrUpdateAd;
+import ru.skypro.homework.dto.ads.ExtendedAd;
 
+@Slf4j
 @RestController
 @RequestMapping("/ads")
 @CrossOrigin(value = "http://localhost:3000")
 @RequiredArgsConstructor
+@Tag(name = "Объявления")
 public class AdsController {
 
+    @GetMapping
+    @Operation(
+            summary = "Получение всех объявлений",
+            responses = {
+                    @ApiResponse(responseCode = "200",
+                    description = "OK",
+                    content = @Content(
+                            mediaType = MediaType.APPLICATION_JSON_VALUE,
+                            schema = @Schema(implementation = Ads.class)
+                    ))
+            }
+    )
+    public ResponseEntity<Ads> getAllAds() {
+       return ResponseEntity.ok(new Ads());
+    }
+
+
     @PostMapping
-    public Ad ad(@RequestBody Ad ad) {
-        return new Ad();
+    @Operation(
+            summary = "Добавление объявления",
+            responses = {
+                    @ApiResponse(responseCode = "201",
+                            description = "Created",
+                            content = @Content(
+                                    mediaType = MediaType.APPLICATION_JSON_VALUE,
+                                    schema = @Schema(implementation = Ad.class)
+                            )),
+                    @ApiResponse(responseCode = "401", description = "Unauthorized", content = @Content(schema = @Schema(hidden = true)))
+            }
+    )
+    public ResponseEntity<Ad> addAd(@RequestPart Ad createdAd) {
```

Comment:
```suggestion
    public ResponseEntity<Ad> addAd(@RequestPart("properties") Ad createdAd, @RequestPart("image") MultipartFile image) {
```
вот так должно быть - названия частей и 2 части, а не 1